rules/PSR4/FileRelocationResolver.php

```diff
@@ -162,7 +162,7 @@ private function replaceRelativeFilePathsWithBeforeAfter(
         // A. first "dir has changed" dummy detection
         $relativeFilePathParts = Strings::split(
             $oldSmartFileInfo->getRelativeFilePath(),
-            '#' . DIRECTORY_SEPARATOR . '#'
```

Comment:
on windows this line translates to `#\#` which is handled by regex as a sharp, followed by a escaped-sharp. therefore the ending delimiter for the regex is missing .. LOL :)

Reply:
Ah, nice one 🥇

Reply:
Could you add comment there to explain? I personally would remove it seeing this

Reply:
done